php/Blockonomics.php

```diff
@@ -521,7 +521,8 @@ public function create_new_order($order_id, $crypto){
                 'order_id'           => $order_id,
                 'status'             => -1,
                 'crypto'             => $crypto,
-                'address'            => $address
+                'address'            => $address,
+                'timestamp'          => time()
         );
```

Comment:
I understand the change here and how it works. However logically it feels weird as same order_id has multiple status and we really don't know whether order has been paid or unpaid

A better structure would be an order and a payments table
Again @DarrenWestwood  feel free to let us know your thoughts

Reply:
> logically it feels weird as same order_id has multiple status

The table name `blockonomics_orders` is causing this confusion. This table should be viewed as a list of `crypto_addresses` generated by Blockonomics and their relevant data. The order_id therefore tells us which woocommerce order the blockonomics address is linked to and the status for the address.

> we really don't know whether order has been paid or unpaid

In the past we were tracking the expected `satoshi` in the db and `paid_<crypto>` amount in the post meta (custom fields). The `paid_<crypto>` amount in the post meta has been adapted in this PR to `paid_order_amount` (fiat) and `paid_to_<address>` (btc). This seems to be functioning well, however satoshi should rather be used or we should save the `satoshi_payed` to the database, and continue displaying btc/fiat in the post meta.

Reply:
Yes I undestand it is crypto_addresses but to the simple question, was the order paid or underpaid is not payed is difficult in the current structure.
The two table structure keeps order payments into a separate table and  is also directly inspired from WHMCS

Reply:
Feel free to create a document with comparsion of the two structures

Reply:
The `satoshi_payed` column should make it easier to see if each address received the correct amount and matches the same structure from WHMCS:
https://docs.google.com/document/d/18aoTcTA6iPERgCGYPtHQI4o8GqutmXS4BYsuaGT8Kgg/edit?usp=sharing